### PR TITLE
[FIO internal] ta: fiovb: bootupgrade_primary_updated variable

### PR DESCRIPTION
**Pull request by igoropaniuk**
Introduce additional variable bootupgrade_primary_updated needed for
handling boot firmware alternative update path
bootupgrade_primary_updated.

Signed-off-by: Igor Opaniuk <igor.opaniuk@foundries.io>

### CODE REVIEW

File: ta/fiovb/include/ta_fiovb.h [hunk 1]
```diff
@@ -10,7 +10,7 @@
 
 #define PERSIST_VALUE_LIST {"bootcount", "upgrade_available", "rollback", \
 			    "m4hash", "m4size", "m4_update", "bootupgrade_available", \
-			    "bootfirmware_version"}
+			    "bootfirmware_version", "bootupgrade_primary_updated"}
```
Comment (ricardosalveti) [inline ta/fiovb/include/ta_fiovb.h:13]: nit, can you move the order to have bootupgrade_primary_updated after bootupgrade_available?

**Review by ricardosalveti — APPROVED** [review]
LGTM other than my minor nit.

**Review by Tim-Anderson — APPROVED** [review]
LGTM

**Comment by mike-scott** [comment]
Definitely address @ricardosalveti comment, but it LGTM.

**Review by mike-scott — APPROVED** [review]

File: ta/fiovb/include/ta_fiovb.h (hunk 1 above)
Comment (igoropaniuk) [reply ta/fiovb/include/ta_fiovb.h]: done

**Comment by igoropaniuk** [comment]
addressed @ricardosalveti comment, please do not merge this till https://github.com/foundriesio/meta-lmp/pull/461 is reviewed/testd

**Pull request merged**